services/searchengine/bleveengine/bleve.go

```diff
@@ -303,7 +304,9 @@ func (b *BleveEngine) UpdateConfig(cfg *model.Config) {
 	b.Mutex.Lock()
 	defer b.Mutex.Unlock()
 
-	mlog.Info("UpdateConf Bleve")
+	if !reflect.DeepEqual(cfg.BleveSettings, b.cfg.BleveSettings) {
+		mlog.Info("UpdateConf Bleve")
+	}
```

Comment:
I think that if we decide that there are no changes as part of the update, we could directly return instead of going through the update logic:

```suggestion
	if reflect.DeepEqual(cfg.BleveSettings, b.cfg.BleveSettings) {
		return
	}

	mlog.Info("UpdateConf Bleve")
```

Reply:
🤦 makes sense 👍